cvat-ui/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.tsx",
   "scripts": {
     "build": "webpack --config ./webpack.config.js",
-    "start": "webpack serve --env API_URL=http://localhost:7000 --config ./webpack.config.js --mode=development",
+    "start": "webpack serve --env API_URL=\"http://${CVAT_HOST:-'localhost'}:7000\" --config ./webpack.config.js --mode=development",
```

Comment:
When is the backend not on localhost (in a development environment)?

Reply:
I setup dev env on VPS and trying to adjust settings to this case

Reply:
If leaving API_URL as `localhost`, it generally works, but at the same time leads to some issues:
e.g. logo URL or dataset export URLs returned with `localhost:7000/../..` what will not work in case with remote development